docs/components/cloud-console/manage-clusters/create-cluster-include.md

```diff
@@ -3,18 +3,18 @@
 
 Let's take a closer look at creating clusters and viewing their details.
 
-[//]:# (Do we need an id and title above?)
+[//]:# (Do we need an id and title above? Seems we're still missing this.)
```

Comment:
I thought this page is this one - https://docs.camunda.io/docs/components/cloud-console/manage-clusters/create-cluster/

But looking through the repo I see we actually have two similar files - 
![image](https://user-images.githubusercontent.com/1051363/153665046-ef8d37f9-e827-47b2-b848-64d4feec5672.png)

Can you check to see if this file is referenced anywhere else? Is it part of sidebars.js? My guess is no. 

Go ahead and delete this page and if the internal link check fails when you go to commit, update the link references to link to `create-cluster.md` instead of `create-cluster-include.md`.

Reply:
I'm confused because we call `import CreateCluster from './create-cluster-include.md'` in the `create-cluster` doc -- should we just combine the two to ensure all of the appropriate information is within `create-cluster.md`?

Reply:
@ultraschuppi @urbanisierung are these pages intentionally separate?

Reply:
Actually @christinaausley if you want to remove that comment and follow up with them separately about those pages, we are good to merge from my point of view. WDYT?

Reply:
Sounds great!